### PR TITLE
feat(console): dark logo

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add dark logo url in sign in experience form.

## Linear Issue Reference

LOG-2332

## Testing

Local tested.

@simeng-li

### CODE REVIEW

File: packages/console/src/pages/SignInExperience/utilities.ts [hunk 1]
```diff
@@ -57,11 +57,19 @@ export const signInExperienceParser = {
...
     } = setup;
 
     return {
       ...setup,
+      branding: {
+        ...branding,
+        // Transform empty string to undefined
+        darkPrimaryColor: branding.darkPrimaryColor?.length ? branding.darkPrimaryColor : undefined,
+        darkLogoUrl: branding.darkLogoUrl?.length ? branding.darkLogoUrl : undefined,
+        slogan: branding.slogan?.length ? branding.slogan : undefined,
```
Comment (demonzoo) [inline packages/console/src/pages/SignInExperience/utilities.ts:71]: Why do we want to check the length first? Would it be equivelant to just
```typescript
const { darkPrimaryColor, darkLogoUrl, slogan } = branding ?? {};

return {
  ...setup,
  branding: { ...branding, darkPrimaryColor, darkLogoUrl, slogan },
}
```

Comment (wangsijie) [reply packages/console/src/pages/SignInExperience/utilities.ts:71]: to remove empty string `''`, or it won't pass zod check in API.

Comment (wangsijie) [reply packages/console/src/pages/SignInExperience/utilities.ts:71]: a `null` or `undefined` value applied to our `input` will become `''`

Comment (wangsijie) [reply packages/console/src/pages/SignInExperience/utilities.ts:71]: the solution is ugly though, any idea?

**Review by demonzoo — APPROVED** [review]

**Review by simeng-li — APPROVED** [review]
cool please ping me once is merged. Just in case I forgot.

**Pull request merged**